build/deploy/generate-deploy-examples.sh

```diff
@@ -0,0 +1,37 @@
+#! /usr/bin/env bash
+
+
+SCRIPT_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)
+MANIFEST_FILE=../../deploy/examples/multus-validation.yaml
+SED=${SCRIPT_ROOT}/../../build/sed-in-place
+
+
+
+run_gen() {
+  if [[ ${SKIP_GEN} = true ]]; then
+    echo "SKIP_GEN is set, skipping manifest  generation"
+    exit
+  fi
+      OUT_FILE="${SCRIPT_ROOT}/${MANIFEST_FILE}"
+      TMP_FILE="$(mktemp)"
+
+ START_MARKER='# THE BELOW HELP TEXT IS AUTO-GENERATED BY `make docs`$'
+ END_MARKER='# THE ABOVE HELP TEXT IS AUTO-GENERATED BY `make docs`$'
+
+
+
+	  go run ${SCRIPT_ROOT}/../../cmd/rook/*.go multus validation run --help | awk '{print "#" $0}' > ${TMP_FILE}
+
+	${SED} -e "/$START_MARKER/,/$END_MARKER/{ /$START_MARKER/{ p ; r $TMP_FILE
+                }; /$END_MARKER/p; d; }" -i.orig ${OUT_FILE}
+
+
+		rm -f ${TMP_FILE}
+
+
+
+
+}
+
+run_gen
+
```

Comment:
Are there 2 newlines at the end of the file? There should be 1.

Reply:
done by applying your suggestion